core/src/main/java/hudson/ExtensionFinder.java

```diff
@@ -758,5 +736,19 @@ private Level logLevel(IndexItem<Extension, Object> item) {
         }
     }
 
+    protected static Class<?> getaClass(IndexItem<Extension, Object> item) throws InstantiationException {
```

Comment:
Not sure it follows the Java naming convention
```suggestion
    protected static Class<?> getAClass(IndexItem<Extension, Object> item) throws InstantiationException {
```

But better with better name
```suggestion
    protected static Class<?> getClassFromIndex(IndexItem<Extension, Object> item) throws InstantiationException {
```

And of course, changing the references :)

Reply:
I'll go w/ the latter

Reply:
#4079